vm-transfer/zvm-transfer.py

```diff
@@ -0,0 +1,494 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+"""
+A python script that clones a VM from the source system to the destination system
+"""
+
+__version__ = "0.1.0"
+
+import os
+import sys
+import logging
+import argparse
+import atexit
+import requests
+from pprint import pformat, pprint
+from munch import unmunchify
+
+sys.path.append('/opt/symphony-client')
+
+import symphony_client
+from config import Config
+
+LOGS_DIR = "/var/log"
+STRATO_LOGS_DIR = LOGS_DIR + "/stratoscale"
+ZVM_CLONE_LOGS_DIR = STRATO_LOGS_DIR + "/zvm-transfer"
+LOGGER_NAME = "zvm-transfer"
+VPSA_VOLUME_TEMPLATE = 'neokarm_volume-{}'
+
+DRY_RUN = Config.DEFAULT_IS_DRY_RUN
+
+logger = logging.getLogger(LOGGER_NAME)
+
+arguments = None
+
+
+def migrate_vm(vm_id_or_name):
+    logger.info("vm_id: %s", vm_id_or_name)
+
+    src_symp_client = init_src_symp_client()
+    vm_list = src_symp_client.vms.list(detailed=True)
+    vm = get_vm_by_id(src_symp_client, vm_id_or_name, vm_list)
+    if not vm:
+        vm = get_vm_by_name(src_symp_client, vm_id_or_name)
+        if not vm:
+            sys.exit(1)
+    vm_id = vm.id
+    if vm.status == 'active':
+        logger.info("VM {} ({}) is active in source cluster".format(vm.name, vm_id))
```

Comment:
There are many possible statuses for a VM (Shutoff, Building, Rescued, Error, Stopped). Maybe it's better to test for the status that we expect. Also, I think it could be helpful to tell the user not only the current VM status, but also the expected VM status.